pangeo_forge_recipes/config.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass, field
+from typing import Optional, Union
+
+from pangeo_forge_recipes.storage import CacheFSSpecTarget, FSSpecTarget
+from pangeo_forge_recipes.transforms import RequiredAtRuntimeDefault
+
+
+@dataclass
+class Config:
+    """a simple class that contains all possible injection spec values
+
+    folks can import it into their recipes to pass injection spec values
+    around to custom beam routines as the example below shows
```

Comment:
Conventionally, our docstrings tend to be a bit less colloquial, could we formalize the language a bit here? Thanks!